### ISSUE TITLE
In-game chat system

### ISSUE THREAD

**Issue by SirPython** [Cardshifter/HTML-Client#113]
This may have been mentioned before (I can't find an issue on it, however), but it would be nice to have an in-game chat system so there was a way you could communicate with your opponent during the game.

**Comment by Phrancis** [comment]
This may not be too hard to do with the new Vue.js client, we would just need to find a way to persist a chat component between page loads, perhaps we could try putting a JSON object in `localStorage` and then loading it into a smaller version of the chat?

One idea is perhaps this mini-chat could be accessed from top navbar at any time (except before login). Let's revisit this after #147 is done.

**Comment by Zomis** [comment]
There might be some components on https://vuetifyjs.com/en/ that could be used for this.
